### PR TITLE
adds images urls and math encodings

### PR DESCRIPTION
**Pull request by dylandechant**

### CODE REVIEW

**Comment by dylandechant** [comment]
@econnally your latest work's export is here...i notice there are some additions. thoughts @econnally @karln ?

**Comment by dylandechant** [comment]
@econnally if you look at the files changed. some files have changes to them (correctly encoded math, image URLs), while others are completely new files.

**Comment by econnally** [comment]
> @econnally if you look at the files changed. some files have changes to them (correctly encoded math, image URLs), while others are completely new files.

@dylandechant are you looking at the zip i posted to notion earlier today? i'm just trying to figure out where to look.

FWIW
* i updated the db so that might have pulled in some new stuff
* i think some of the exports include things that weren't there before (e.g. activity components?)

**Comment by econnally** [comment]
> > @econnally if you look at the files changed. some files have changes to them (correctly encoded math, image URLs), while others are completely new files.
> 
> @dylandechant are you looking at the zip i posted to notion earlier today? i'm just trying to figure out where to look.
> 
> FWIW
> 
> * i updated the db so that might have pulled in some new stuff
> * i think some of the exports include things that weren't there before (e.g. activity components?)

sorry i'm being dense i thought this was an old PR. the diffs you mean ARE this PR

**Comment by dylandechant** [comment]
@econnally this PR is a diff between what we delivered on the 5th of may (deadline), and this new delivery (adding image URLs, and correctly encoding math).  So, this PR is taking your export this morning and merging it into the export from the 5th.

**Comment by econnally** [comment]
> @econnally this PR is a diff between what we delivered on the 5th of may (deadline), and this new delivery (adding image URLs, and correctly encoding math). So, this PR is taking your export this morning and merging it into the export from the 5th.

@dylandechant now i see. that is a pretty big diff. i can see why you are concerned.

File: build/05-05/cool-down-node-465179.ocx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <div class="Assessment.name">Train Speeds</div>
+    <div class="im_statement"><p>Two trains are traveling at constant speeds on different tracks.</p>
+
+<p>Train A:</p>
+
+<p><img src="/tikz_files/205275.svg" /></p>
+
+<p>Train B:</p>
+
+<p><img src="/tikz_files/205276.svg" /></p>
```
Comment (dylandechant) [inline build/05-05/cool-down-node-465179.ocx:82]: @econnally looks like we are missing image URL resolutions for tikz files too...

**Review by dylandechant — COMMENTED** [review]
@econnally can you look at the one comment I have on here...

File: build/05-05/cool-down-node-465179.ocx (hunk 1 above)
Comment (econnally) [reply build/05-05/cool-down-node-465179.ocx:82]: @dylandechant watching pairing sessions now. i'll look at this tomorrow unless you tell me it's a red alert

**Pull request closed without merging**